docs/_documentations/importing-existing-projects.md

```diff
@@ -164,10 +164,10 @@ FROM ibmjava:8-sfj
 
 Your `package.json` must meet the following requirements. For an example of a good `package.json` for importing, see [the express application generator](https://github.com/ibm-developer/generator-ibm-core-node-express/blob/develop/app/templates/package.json):
 
-- Ensure the project provides a `start` npm script in `package.json` so it can be started by `npm start`.
-    - For example: `start: "node server/server.js"`
-- In local installations of Codewind, you can restart the project in Debug mode. To use this feature, the project must also provide a `debug` npm script, which accepts connections from all hosts on port 9229. For help configuring this script, see [the Node.js debugging guide](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options).
-    - For example: `debug: "node --inspect=0.0.0.0:9229 server/server.js"`
+- Ensure the project provides a `start` npm script in `package.json`, so it can be started by `npm start`.
+    - For example: `start: "node server/server.js"`.
+- In local installations of Codewind, you can restart the project in Debug mode. To use this feature, the project must also provide a `debug` npm script which accepts connections from all hosts on port 9229. For help configuring this script, see [the Node.js debugging guide](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options).
```

Comment:
Comma needed before "which" to set off grammatically nonessential clause. Or, and this options is probably even better, replace "which" with "that."